spk/fossil-scm/Makefile

```diff
@@ -0,0 +1,22 @@
+SPK_NAME = fossil-scm
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/fossil-scm.png
+
+DEPENDS = cross/fossil-scm
+
+MAINTAINER = SynoCommunity
+DISPLAY_NAME = Fossil SCM
+
+OPTIONAL_DESC =
```

Comment:
remove this variable as it is not used here (I see, you cloned a synocli-* Makefile)